xbmc/video/dialogs/GUIDialogVideoBookmarks.cpp

```diff
@@ -510,14 +495,22 @@ CGUIControl *CGUIDialogVideoBookmarks::GetFirstFocusableControl(int id)
   return CGUIWindow::GetFirstFocusableControl(id);
 }
 
-bool CGUIDialogVideoBookmarks::AddEpisodeBookmark()
+bool CGUIDialogVideoBookmarks::GetEpisodesByFile(const std::string& currentFile, vector<CVideoInfoTag>& episodes)
 {
-  vector<CVideoInfoTag> episodes;
   CVideoDatabase videoDatabase;
   videoDatabase.Open();
-  videoDatabase.GetEpisodesByFile(g_application.CurrentFile(), episodes);
+  videoDatabase.GetEpisodesByFile(currentFile, episodes);
   videoDatabase.Close();
-  if(episodes.size() > 0)
+  return episodes.size() > 1;
+}
+
+bool CGUIDialogVideoBookmarks::AddMediaBookmark()
+{
+  if (!g_application.CurrentFileItem().HasVideoInfoTag() || MediaTypes::IsCollection(g_application.CurrentFileItem().GetVideoInfoTag()->m_type))
+    return false;
+  vector<CVideoInfoTag> episodes;
```

Comment:
Please add an empty line between the return and this line for better readability.